packages/core/components/AnnotationPicker/index.tsx

```diff
@@ -4,8 +4,9 @@ import { useSelector } from "react-redux";
 import ListPicker from "../ListPicker";
 import { ListItem } from "../ListPicker/ListRow";
 import { TOP_LEVEL_FILE_ANNOTATION_NAMES } from "../../constants";
-import Annotation from "../../entity/Annotation";
-import { metadata, selection } from "../../state";
+import Annotation, { AnnotationName } from "../../entity/Annotation";
+import { selection } from "../../state";
+import { uniqBy } from "lodash";
 
 interface Props {
     disabledTopLevelAnnotations?: boolean;
```

Comment:
Line 14: outside the scope of this PR, but I'm unsure if the `disableUnavailableAnnotations` prop still does anything?